### PR TITLE
Namespace crypt

### PR DESCRIPTION
**Pull request by laoneo**
Namespace crypt. I'm aware that some classes will be skipped in J4, but for consistency I namespaced all.

### CODE REVIEW

File: libraries/src/Joomla/CMS/Crypt/CryptKey.php [hunk 1]
```diff
@@ -16,7 +17,7 @@
  *
  * @since  12.1
  */
-class JCryptKey
+class CryptKey
```
Comment (mbabker) [inline libraries/src/Joomla/CMS/Crypt/CryptKey.php]: For consistency with the Framework can we just name this `Key` instead?

File: libraries/src/Joomla/CMS/Crypt/Cipher/TrippleDesCipher.php [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 
 /**
  * JCrypt cipher for Triple DES encryption, decryption and key generation.
  *
  * @since       12.1
- * @deprecated  4.0   Without replacement use JCryptCipherCrypto
+ * @deprecated  4.0   Without replacement use CryptoCipher
  */
-class JCryptCipher3Des extends JCryptCipherMcrypt
+class TrippleDesCipher extends McryptCipher
```
Comment (mbabker) [inline libraries/src/Joomla/CMS/Crypt/Cipher/TrippleDesCipher.php]: Triple only has one "p" in it.

File: libraries/classmap.php [hunk 3]
```diff
@@ -294,3 +294,15 @@
 JLoader::registerAlias('JFTP',                              '\\Joomla\\CMS\\Client\\FtpClient', '4.0');
 JLoader::registerAlias('JClientLdap',                       '\\Joomla\\Ldap\\LdapClient', '5.0');
 JLoader::registerAlias('JLDAP',                             '\\Joomla\\Ldap\\LdapClient', '4.0');
+
+JLoader::registerAlias('JCrypt',                             '\\Joomla\\CMS\\Crypt\\Crypt', '5.0');
+JLoader::registerAlias('JCryptCipher',                       '\\Joomla\\CMS\\Crypt\\CipherInterface', '5.0');
+JLoader::registerAlias('JCryptKey',                          '\\Joomla\\CMS\\Crypt\\CryptKey', '5.0');
```
Comment (mbabker) [inline libraries/classmap.php]: Need to fix the alias.

**Comment by laoneo** [comment]
Done

**Pull request merged**